.gitignore

```diff
@@ -26,6 +26,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+src/aind_metadata_mapper/bruker/MRI_ingest/MRI_files
```

Comment:
Can you remove this line? Any files you are adding can be added to a folder that is already in the .gitignore